budget/utils.py

```diff
@@ -86,25 +88,33 @@ def list_of_dicts2json(dict_to_convert):
     """Take a list of dictionnaries and turns it into
     a json in-memory file
     """
-    bytes_io = BytesIO()
-    bytes_io.write(dumps(dict_to_convert))
-    bytes_io.seek(0)
-    return bytes_io
+    return BytesIO(dumps(dict_to_convert).encode('utf-8'))
 
 def list_of_dicts2csv(dict_to_convert):
     """Take a list of dictionnaries and turns it into
     a csv in-memory file, assume all dict have the same keys
     """
-    bytes_io = BytesIO()
+    # CSV writer has a different behavior in PY2 and PY3
```

Comment:
That's not true if you are opening the file with 'rb' or 'wb', we should use BytesIO in both cases.